repo/packages/C/cassandra/700/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "packagingVersion": "4.0",
+  "upgradesFrom": [
+    "2.6.0-3.11.4"
```

Comment:
Can this version also be upgraded/downgraded from the previous version `2.5.0-3.11.3`? I would think so since base tech is in the same minor version - can you test it?

Reply:
I dont think so that would be by default, i think we will have to configure for that. 
let me check it !!

Reply:
no currently 2.5.0-3.11.3 doesnt pick 2.7.0-3.11.4 for upgraded/downgraded from.